

 
     public ContainerBrewingStand(int i, PlayerInventory playerinventory, IInventory iinventory, IContainerProperties icontainerproperties) {
         super(Containers.BREWING_STAND, i);
+        player = playerinventory; // CraftBukkit
         checkContainerSize(iinventory, 5);
         checkContainerDataCount(icontainerproperties, 2);
         this.brewingStand = iinventory;
@@ -61,6 +72,7 @@
 
     @Override
     public boolean stillValid(EntityHuman entityhuman) {
+        if (!this.checkReachable) return true; // CraftBukkit
         return this.brewingStand.stillValid(entityhuman);
     }
 
@@ -198,4 +210,17 @@
